### DISCUSSION TITLE
egzumer .21 - is it possible to access memory channels directly with key access

### DISCUSSION THREAD

**Discussion by tjcirillo** [egzumer/uv-k5-firmware-custom#393] [Q&A]

**Comment by tjcirillo** [comment]
when i try to enter a memory location directly, it accepts the two digits and waits for something like an enter key, which is not there, and then it times out.   it's as though the function is actually programmed but isn't working?

**Reply by JuantAldea** [reply]
Start the number with a zero.

**Reply by tjcirillo** [reply]
Thank you

**Comment by tjcirillo** [comment]
operator error; ignore
